funding/manager.go

```diff
@@ -104,14 +104,19 @@ type Manager struct {
 	// BatchStepTimeout is the timeout the manager uses when executing a
 	// single batch step.
 	BatchStepTimeout time.Duration
+
+	// NotifyShimCreated is a function that should be called whenever a
+	// funding shim is created for a bid order where we expect an incoming
+	// channel at any moment.
+	NotifyShimCreated func(ourBid *order.Bid, pendingChanID [32]byte)
```

Comment:
Alternatively, we could've extended the funding shim to also verify this information just as it does the pubkey/amt, etc. One other upcoming use case for this would be once we make channel types explicit in the protocol, and allow users to purchase precise channel types in Pool.

Reply:
Yes, I think we should do this in 0.13 and then also add a bunch more of our RPCs we use to `lndclient` so we can also get rid of the `baseClient` variable we have in there. That way, after 0.13, we can do a "cleanup" PR for Pool and the server to simplify some things.